tests/machine.test.ts

```diff
@@ -53,15 +54,28 @@ describe("Versions", () => {
     });
 });
 
+jest.setTimeout(15000);
+
 describe(OlmMachine.name, () => {
     test("can be instantiated with the async initializer", async () => {
         expect(await OlmMachine.initialize(new UserId("@foo:bar.org"), new DeviceId("baz"))).toBeInstanceOf(OlmMachine);
     });
 
-    test("can be instantiated with a store", async () => {
+    test("can be instantiated with a StoreHandle", async () => {
         let storeName = "hello";
         let storePassphrase = "world";
 
+        let storeHandle = await StoreHandle.open(storeName, storePassphrase);
+        expect(
+            await OlmMachine.init_from_store(new UserId("@foo:bar.org"), new DeviceId("baz"), storeHandle),
+        ).toBeInstanceOf(OlmMachine);
+        storeHandle.free();
```

Comment:
Is it needed to call it explicitly?

Reply:
There are no concept of deterministic finalizers in JavaScript (the only finalizers that exist are non-deterministic due to the nature of GC).

Reply:
Yeah. In theory the finalizer should get run soon after `storeHandle` goes out of scope; in practice it seemed to take a while for the garbage collector to clean it up. Without this, the tests ran slowly.